app/views/layouts/application.html.erb

```diff
@@ -1,30 +1,26 @@
 <!DOCTYPE html>
 <html>
+<head>
+  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
```

Comment:
Diese sollten über Assets reingehen.

Reply:
Das ist aus dem Main

Reply:
Was ist aus dem Main?
Schaue bitte hier, das nutzt du hiermit gar nicht.
https://edgeguides.rubyonrails.org/asset_pipeline.html
https://edgeguides.rubyonrails.org/webpacker.html

Musst du aber!

Reply:
Das ist nicht aus einen von meinen PR, sonder aus dem https://github.com/Ktechen/Dripbox/pull/15

Reply:
Das ist unerheblich, da es zum Bereich Webprojektmanagement gehört und unter allen im Team verteilt wird. Ich habe es bereits in #15 angesprochen, wurde aber vor dem Merge nicht behoben. Ihr müsst das als Team klären.

Reply:
Achso, okay

Reply:
Behebt das bitte hier. Richtigen Setup für Bootstrap 5 und nicht für 4 und benutzt bitte dann auch 5. Das haben wir am Anfang des Semester besprochen.

Reply:
Ja, wie schon gesagt habe das ja nicht hinzugefügt und habe @MathisHTW bescheid gesagt das er das fixen soll.